Description: do not use make to extract version
 Without a valid config.mk, trying to call m4_esyscmd_s() to run
 make results in weird error messages. This patch avoids having
 to run make just to extract this value.
Author: Sascha Steinbiss <sascha@steinbiss.name>
--- a/configure.ac
+++ b/configure.ac
@@ -23,7 +23,7 @@
 # DEALINGS IN THE SOFTWARE.
 
 dnl Process this file with autoconf to produce a configure script
-AC_INIT([HTSlib], m4_esyscmd_s([make print-version]),
+AC_INIT([HTSlib], m4_esyscmd_s([grep 'PACKAGE_VERSION  =' Makefile | cut -f 2 -d'=' | xargs]),
         [samtools-help@lists.sourceforge.net], [], [http://www.htslib.org/])
 AC_PREREQ(2.63)  dnl This version introduced 4-argument AC_CHECK_HEADER
 AC_CONFIG_SRCDIR(hts.c)
